src/routes/AppRouter.js

```diff
@@ -1,15 +1,108 @@
-import { Route, Routes } from "react-router-dom"
-import EyeAccountCreationPage from "../pages/eyeAccount/EyeAccountCreationPage"
-import EyeAccountListingPage from "../pages/eyeAccount/EyeAcountListingPage"
+import { Route, Routes } from "react-router-dom";
+import EyeAccountCreationPage from "../pages/eyeAccount/EyeAccountCreationPage";
+import EyeAccountListingPage from "../pages/eyeAccount/EyeAcountListingPage";
 
-const  AppRouter = () =>  {
+// routes
+const routes = [
+  {
+    path: "*",
+    exact: true,
+    element: () => "Page Not Found",
+  },
+  {
+    path: "/",
+    exact: true,
+    element: () => "Hub page TBC",
+  },
+  {
+    path: "/dashboard",
+    exact: true,
+    element: () => "Dashboard",
+  },
+  {
+    path: "/eyeAccount",
+    exact: true,
+    element: () => <EyeAccountListingPage />,
+  },
+  {
+    path: "/headmate-meeting-space",
+    exact: true,
+    element: () => "Headmate Meeting Space",
+  },
+  {
+    path: "/system-map",
+    exact: true,
+    element: () => "System Map",
+  },
+  {
+    path: "/expressway",
+    exact: true,
+    element: () => "Expressway",
+  },
+  {
+    path: "/reminders",
+    exact: true,
+    element: () => "Reminders",
+  },
+  {
+    path: "/system-rules",
+    exact: true,
+    element: () => "body1" > "System Rules",
+  },
+  {
+    path: "/appreciation-station",
+    exact: true,
+    element: () => "Appreciation Station",
+  },
+  {
+    path: "/treatment-hub",
+    exact: true,
+    element: () => "Treatment Hub",
+  },
+  {
+    path: "/resources",
+    exact: true,
+    element: () => "Resources",
+  },
+  {
+    path: "/guide",
+    exact: true,
+    element: () => "Guide",
+  },
+  {
+    path: "/settings",
+    exact: true,
+    element: () => "Settings",
+  },
+  {
+    path: "/leave-a-review",
+    exact: true,
+    element: () => "Leave a Review",
+  },
+  {
+    path: "/sign-out",
+    exact: true,
+    element: () => "Sign Out",
+  },
+  {
+    path: "/eyeAccount/new",
+    exact: true,
+    element: () => <EyeAccountCreationPage />,
+  },
+];
+
+const AppRouter = () => {
   return (
     <Routes>
-        <Route path='/' element={"Hub page TBC"} />
-        <Route path='/eyeAccount' element={<EyeAccountListingPage/>} />
-        <Route path='/eyeAccount/new' element={<EyeAccountCreationPage/>} />
+      {routes.map((route, index) => (
+        <Route path={route.path} element={route.element()} key={index} />
```

Comment:
Very minor one with this one. Might be worth making the key something other than index here. Not that this shoule ever really be an issue but probably best to use the `route.path` for this.

See here as to why: `https://reactjs.org/docs/lists-and-keys.html#keys`